src/otx/algorithms/common/configs/training_base.py

```diff
@@ -198,6 +198,17 @@ class BaseLearningParameters(ParameterGroup):
             affects_outcome_of=ModelLifecycle.TRAINING,
         )
 
+        input_size = selectable(
+            default_value=InputSizePreset.DEFAULT,
+            header="Configure model input size.",
+            description="The input size of the given model could be configured to one of the predefined resolutions."
+            "Reduced training and inference time could be expected by using smaller input size."
+            "Defaults to per-model default resolution.",
+            warning="Modifying input size may decrease model performance.",
```

Comment:
Same here :)

Reply:
Actually I changed it as Sungman said after applying your feedback. He said that user not just decrease input size but also increase and I agree on that. How do you think about that?